blockrecord/header.go

```diff
@@ -77,6 +78,14 @@ type Header struct {
 	Nonce            NonceType              `json:"nonce"`
 }
 
+var log *logger.L
+
+// Initialise - initialize logger
```

Comment:
I think this method does not intend to initialize the logger only, right? The name `Initialise` makes me think it's defined to initialized the header. If it's just for initializing the logger so the name should be `InitialiseLogger()`, if it does not intend to initialize the logger only, you should correct the comment

Reply:
i follow the convention, every other module with Initialise function

Reply:
Yes. I know but i am concern about the comment `initialize logger`. It should not stand for initializing the logger only if the method name is `Initialise`.

Reply:
ok, modify comment